internal/utils/config.go

```diff
@@ -36,6 +36,21 @@ var (
 	VectorId      string
 	PoolerId      string
 
+	DbAliases          = []string{"db"}
+	KongAliases        = []string{"kong", "api"}
```

Comment:
Alternatively we could make this something like:
```suggestion
	KongAliases        = []string{"kong", "api.supabase.internal"}
```